templates/cli-config.toml.epp

```diff
@@ -8,6 +8,7 @@
   Optional[Boolean]         $verify_ssl = undef,
   Optional[Boolean]         $dry_run = undef,
 | -%>
+# This file is managed by puppet-pulpcore. Manual edits will be overwritten with the next run.
```

Comment:
This may not be true. There's a condition on it being managed:
https://github.com/theforeman/puppet-pulpcore/blob/cb95de666838018aa88c5aba92cd39cb3835ec9b/manifests/cli.pp#L61

So what users can do is:
* Run `foreman-installer`
* Set `pulpcore::manage_root_config: false` in `/etc/foreman-installer/custom-hiera.yaml`
* Run `foreman-installer`

Now you have the file `/root/.config/pulp` on disk with this warning, but it's no longer managed.

I don't see a good way around this.

Reply:
Hmm, is it possible to have Puppet check for just the one line and remove it if `pulpcore::manage_root_config: false`?

Reply:
I have amended the wording.